slim/prune/MODEL_ZOO.md

```diff
@@ -0,0 +1,54 @@
+# 裁剪模型库
```

Comment:
1. move this doc to slim/README.md
2. Also needs to add quant and  distillation model

Reply:
Done, Thanks